Content.Shared/Hands/EntitySystems/SharedHandsSystem.Interactions.cs

```diff
@@ -198,10 +199,20 @@ private void OnGetUsedEntity(EntityUid uid, HandsComponent component, ref GetUse
     //TODO: Actually shows all items/clothing/etc.
     private void HandleExamined(EntityUid examinedUid, HandsComponent handsComp, ExaminedEvent args)
     {
+        // Floof: show carried entities
         var heldItemNames = EnumerateHeld(examinedUid, handsComp)
-            .Where(entity => !HasComp<VirtualItemComponent>(entity))
-            .Select(item => FormattedMessage.EscapeText(Identity.Name(item, EntityManager)))
-            .Select(itemName => Loc.GetString("comp-hands-examine-wrapper", ("item", itemName)))
+            .Select(entity => new Entity<VirtualItemComponent?>(entity, CompOrNull<VirtualItemComponent>(entity)))
+            .Where(entity => entity.Comp is null
+                // We can't access CarryingSystem in Shared, so here's a hacky way to check if we're carrying the object.
+                // A carried object is a child of the carrier, and is not a set of handcuffs on them.
+                || TransformSystem.GetParentUid(entity.Comp.BlockingEntity) == examinedUid
+                && !(TryComp(examinedUid, out CuffableComponent? cuffable)
+                    && cuffable.Container.ContainedEntities.Contains(entity.Comp.BlockingEntity)))
+            .Select(item => item.Comp?.BlockingEntity ?? item)
```

Comment:
Not a terrible way to do this, but why not just do
```suggestion
            .Select(entity => new Entity<VirtualItemComponent?>(entity, CompOrNull<VirtualItemComponent>(entity)))
            .Where(entity => entity.Comp is null || entity.Comp.BlockingEntity != examinedUid)
            .Select(item => item.Comp?.BlockingEntity ?? item)
```
? This would also show pulled entities and perhaps more. I don't think it's a bad idea to say that the mob is "holding handcuffs" while they are cuffed, either. This would help check if someone is still cuffed without opening the strip menu on them and without looking at their sprite.

Reply:
i was already thinking about how to do the cuffs in a nice way, as well as dragged. i'm open to the idea but i don't want to use the same text as holding because it won't differentiate between someone actually being cuffed vs holding cuffs and cause confusion.

Reply:
> i was already thinking about how to do the cuffs in a nice way, as well as dragged. i'm open to the idea but i don't want to use the same text as holding because it won't differentiate between someone actually being cuffed vs holding cuffs and cause confusion.

I mean, if it's a big deal for you, you could add a new string or enum field to VirtualItemComponent that would indicate the type of the virtual item. Then you could change the code below to fetch slightly different strings based on the type of the virtual item (with the default one - if a type is not set - being "they are holding X"). But in my opinion, the existing text is just fine.